### ISSUE TITLE
Non-profit org?

### ISSUE THREAD

**Issue by joelmccracken** [pghfp/tracking#9]
There are some costs associated with PGHFP (for example, meetup.com).

How do we want to handle the finances?

**Comment by joelmccracken** [comment]
how handle donations on meetup?
“kickstarter” model for costs?

**Comment by bwongchaowart** [comment]
@joelmccracken Incorporating a nonprofit organization and applying for tax-exempt status costs hundreds of dollars and a lot of time. Unless you want to raise thousands of dollars, it's probably not worth it. Are you proposing to form an [unincorporated nonprofit association](https://www.thebalance.com/what-is-an-unincorporated-nonprofit-association-2501895)?

**Comment by joelmccracken** [comment]
I forget all the details, but is something that are in my "pghfp" notes. I walked to @sjfloat about it a while ago, he knows the details I think.  This is really just a placeholder. I think that if we start taking donations or doing anything financial, that gets tricky.

**Comment by sjfloat** [comment]
Yeah, I'd like to keep things simple. But I wouldn't mind offsetting some of the cost and maybe having a little petty cash for incidentals.

**Comment by hanjiexi** [comment]
When it comes time to deal with the complexity of donations/finances, maybe this could help: https://opencollective.com

You can apply as a Meetup Group here: https://opencollective.com/apply

**Comment by sjfloat** [comment]
Thanks. That looks really interesting.

**Comment by sjfloat** [comment]
I don't think we'll likely do a nonprofit thing, but this is still an interesting thread. Might we generalize the issue name?

**Comment by joelmccracken** [comment]
changed it. you can probably change it to a better name though =D

**Comment by sjfloat** [comment]
Perfect

**Comment by joelmccracken** [comment]
this has come up again. basically, we want to host remote-friendly events, and the only real way to do it is to purchase zoom, so we want to discuss how to fund this long term
